custom_components/tuya_local/devices/arlec_smart_button.yaml

```diff
@@ -0,0 +1,28 @@
+name: Smart Button
+products:
+  - id: kpzc6pm8
+    name: Arlec Smart Button
+primary_entity:
+  entity: event
+  name: Button
```

Comment:
The entity should not need a name if the desired name matches the class, generally classes define a default translated name.